liquibase-core/src/main/java/liquibase/command/CommandScope.java

```diff
@@ -89,6 +89,10 @@ public <T> CommandScope addArgumentValue(CommandArgumentDefinition<T> argument,
         return this;
     }
 
+    public SortedMap<String, Object> getArgumentValues() {
```

Comment:
I don't think it's a good idea to expose this private variable. It's not something that we want directly accessible by external clients as it would expose some internal implementation details and lets them go around the correct way of doing things and/or get unexpected results thinking that getArgumentValues() is a list of all the configured arguments and/or the correct values for them. 

I think you need to change your addAll() call below to loop through the flowCommandArguments and call `addArgumentValue()` for each.

Reply:
Nathan and I talked about this offline. He gave me some good ideas, and we basically landed on creating a new value provider that provides the values from the mojo, which will allow downstream usage in the pro code very easily.